waterlog/src/Stylesheets/_progressBar.scss

```diff
@@ -5,44 +5,15 @@
   margin: 0 10px;
 }
 
-.filler-low {
-  background-color: #FFEA00 ;
+.filler {
+  background-color: red;
```

Comment:
use rgb like the other places have

Reply:
Sure thanks, My group member is working on that.